react/features/prejoin/components/preview/DeviceStatus.tsx

```diff
@@ -1,18 +1,21 @@
-// @flow
-
 import { makeStyles } from '@material-ui/styles';
 import clsx from 'clsx';
 import React from 'react';
+import { WithTranslation } from 'react-i18next';
 
-import { translate } from '../../../base/i18n';
-import { Icon, IconCheckSolid, IconExclamationTriangle } from '../../../base/icons';
-import { connect } from '../../../base/redux';
+import { IState } from '../../../app/types';
+import { translate } from '../../../base/i18n/functions';
+import Icon from '../../../base/icons/components/Icon';
+import { IconCheckSolid, IconExclamationTriangle } from '../../../base/icons/svg/index';
+import { connect } from '../../../base/redux/functions';
 import {
     getDeviceStatusType,
     getDeviceStatusText
+
+    // @ts-ignore
 } from '../../functions';
 
-export type Props = {
+export type Props = WithTranslation & {
```

Comment:
nit: a bit of inconsistency, in other places you have interface Props extends WithTranslation